### PR TITLE
update to version 1.0.1

### PR DESCRIPTION
**Pull request by katietz**
* update to version 1.0.1
* added pip, wheel, setuptools as run dependency

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,15 +10,23 @@ source:
...
+  # See conda-forge/glib-feedstock#40 for documentation regarding the RPATH patch
+  patches:
+    - 0001-Only-fix-RPATH-if-install_rpath-is-not-empty.patch
 
 build:
   number: 0
-  skip: True # [py<37]
-  script: {{ PYTHON }} -m pip install . -vv --no-deps
+  script: {{ PYTHON }} -m pip install . -vvv --no-deps
+  skip: True  # [py<36]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:20]: It supports python >=3.7 https://github.com/mesonbuild/meson/blob/f341097c7e7914c250133bd58feb4985d1fbdcd6/setup.py#L19

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,15 +10,23 @@ source:
...
   entry_points:
     - meson = mesonbuild.mesonmain:main
 
+
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:24]: Redundant empty line

File: recipe/meta.yaml [hunk 3]
```diff
@@ -10,15 +10,23 @@ source:
...
 requirements:
+  build:
+    - patch     # [not win]
+    - m2-patch  # [win]
+
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:29]: Redundant empty line

File: recipe/meta.yaml (hunk 1 above)
Comment (katietz) [reply recipe/meta.yaml:20]: hmm, I checked what conda-forge sees as compatible.  But sure, we can go without 3.6

File: recipe/meta.yaml (hunk 2 above)
Comment (katietz) [reply recipe/meta.yaml]: addressed both. thanks!

File: recipe/LICENSE
Comment (M-Waszkiewicz-Anaconda) [inline recipe/LICENSE:1]: what are the rules for the license in recipe folder, I see that you copied from conda forge but this license differs from the one in main project directory, according to https://opensource.stackexchange.com/questions/13172/having-one-sub-folder-in-a-github-repo-under-a-different-license-than-the-whole 
shall we add to main project license such statement 

 "This license applies to the entire repo except for subfolders that have their own license file. In such cases, the license file in the subfolder takes precedence.""

Comment (katietz) [reply recipe/LICENSE:1]: well, no clear idea here.  I think we should follow here CF. but this is my POV

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Pull request merged**